services/migrations/gitea_uploader.go

```diff
@@ -134,12 +134,28 @@ func (g *GiteaLocalUploader) CreateRepo(repo *base.Repository, opts base.Migrate
 		Releases:       opts.Releases, // if didn't get releases, then sync them from tags
 		MirrorInterval: opts.MirrorInterval,
 	}, NewMigrationHTTPTransport())
+	if err != nil {
+		return err
+	}
 
-	g.sameApp = strings.HasPrefix(repo.OriginalURL, setting.AppURL)
-	g.repo = r
+	gitRepo, err := gitrepo.OpenRepository(g.ctx, r)
```

Comment:
It doesn't seem right. There is another `g.gitRepo, err = gitrepo.OpenRepository(g.ctx, r)` below